pkg/operators/common/kubeinventorycache.go

```diff
@@ -53,43 +49,32 @@ var (
 
 func GetK8sInventoryCache() (*K8sInventoryCache, error) {
 	once.Do(func() {
-		cache, err = newCache(1 * time.Second)
+		cache, err = newCache(10 * time.Minute)
 	})
 	return cache, err
 }
 
-func newCache(tickerDuration time.Duration) (*K8sInventoryCache, error) {
+func newCache(defaultResync time.Duration) (*K8sInventoryCache, error) {
 	clientset, err := k8sutil.NewClientset("")
 	if err != nil {
 		return nil, fmt.Errorf("creating new k8s clientset: %w", err)
 	}
+	factory := informers.NewSharedInformerFactory(clientset, defaultResync)
```

Comment:
Do we need a resync period at all? (Really a question, don't have the experience with K8s informers)

Reply:
In most cases it's not needed, I can remove it.

Reply:
Doesn't need to be changed - was just curious